src/solvers/smt2/smt2_dec.h

```diff
@@ -10,10 +10,12 @@ Author: Daniel Kroening, kroening@kroening.com
 #ifndef CPROVER_SOLVERS_SMT2_SMT2_DEC_H
 #define CPROVER_SOLVERS_SMT2_SMT2_DEC_H
 
-#include <fstream>
-
 #include "smt2_conv.h"
 
+#include <util/message.h>
+
+#include <fstream>
```

Comment:
See above for why I think changing the sequence of directories isn't desired. But we might as well want to document one or the other in `CODING_STANDARD.md`.

Reply:
I believe this particular change is consistent with the idea that you want to guard against missing dependencies.

Reply:
I agree with this, and quite possibly we should apply this consistently whenever we make changes to includes - it may be good to document this in `CODING_STANDARD.md` (which currently only talks about the first include.